src/Core/cldj_fjx_sub_mod.F90

```diff
@@ -215,12 +215,19 @@ subroutine PHOTO_JX (U0,SZA,RFL,SOLF, LPRTJ, PPP,ZZZ,TTT,HHH,    &
 
 !-----------------------------------------------------------------------
 
+      ! Initialize location and outputs
+      thisloc = ' -> at PHOTO_JX in module cldj_fjx_sub_mod.F90'
+      rc = CLDJ_SUCCESS
+      VALJXX = 0.d0
```

Comment:
Thanks for initializing the variables.  This will prevent numerical noise errors caused by uninitialized variables.